tools/install-source/Makefile

```diff
@@ -8,23 +9,65 @@ ifdef INCLUDE_MAC
 install-local:: install-source-mac
 endif
 
+MDB_REBASE_EXE = $(abspath $(MONO_PATH))mcs/class/lib/xammac/mdbrebase.exe
+MDB_REBASE = MONO_PATH=$(dir $(MDB_REBASE_EXE)) $(TOP)/builds/install/tools64/bin/mono --debug $(MDB_REBASE_EXE)
```

Comment:
It doesn't look like you're using these two `MDB_REBASE` Make variables anywhere, in which case they can be removed.